docs/USAGE-GUIDE.md

```diff
@@ -239,63 +237,61 @@ s2n-tls usually prefers AES over ChaCha20. However, some clients-- particularly
-|   "20150306"   |      X       |     X    |      X        |          |
-|   "20150214"   |      X       |     X    |      X        |          |
-|   "20150202"   |      X       |     X    |      X        |          |
-|   "20141001"   |      X       |     X    |      X        |          |
-|   "20140601"   |      X       |     X    |      X        |          |
-|   "20190120"   |      X       |     X    |      X        |          |
-|   "20190121"   |      X       |     X    |      X        |          |
-|   "20190122"   |      X       |     X    |      X        |          |
-|   "20190801"   |      X       |     X    |      X        |    X     |
-|   "20190802"   |      X       |     X    |      X        |    X     |
-|   "20200207"   |      X       |     X    |      X        |    X     |
-|   "20230317"   |      X       |     X    |               |    X     |
-|   "rfc9151"    |      X       |     X    |               |    X     |
-
-Note that the default_tls13 security policy will never support legacy SHA-1 algorithms in TLS1.3, but will support
-legacy SHA-1 algorithms in CertificateVerify messages if TLS1.2 has been negotiated.
+|    version    | RSA PKCS1 | ECDSA | SHA-1 Legacy | RSA PSS |
+|---------------|-----------|-------|--------------|---------|
+|   20230317    |     X     |   X   |              |    X    |
+|    default    |     X     |       |       X      |         |
+| default_tls13 |     X     |   X   |       X      |    X    |
+| default_fips  |     X     |   X   |              |         |
+|   20190214    |     X     |   X   |       X      |         |
+|   20170718    |     X     |       |       X      |         |
+|   20170405    |     X     |       |       X      |         |
+|   20170328    |     X     |       |       X      |         |
+|   20170210    |     X     |       |       X      |         |
+|   20160824    |     X     |       |       X      |         |
+|   20160804    |     X     |       |       X      |         |
+|   20160411    |     X     |       |       X      |         |
+|   20150306    |     X     |       |       X      |         |
+|   20150214    |     X     |       |       X      |         |
+|   20150202    |     X     |       |       X      |         |
+|   20141001    |     X     |       |       X      |         |
+|   20190120    |     X     |       |       X      |         |
+|   20190121    |     X     |       |       X      |         |
+|   20190122    |     X     |       |       X      |         |
+|   20190801    |     X     |   X   |       X      |    X    |
+|   20190802    |     X     |   X   |       X      |    X    |
+|   20200207    |           |   X   |              |    X    |
+|    rfc9151    |     X     |   X   |              |    X    |
+
+Note that the legacy SHA-1 algorithms are not supported in TLS1.3. Legacy SHA-1 algorithms will be supported in CertificateVerify messages only if TLS1.2 has been negotiated and the security policy allows them.
 
```

Comment:
Oh one other error here I missed-- it's only consistently CertificateVerify in TLS1.3. In TLS1.2 it's KeyExchange for server auth and CertificateVerify for client auth. We might want to just remove the "in CertificateVerify message" part.